python/lsst/dax/data_generator/generator.py

```diff
@@ -55,6 +56,7 @@ def __init__(self, spec):
 
         self.spec = spec
         self.tables = spec.keys()
```

Comment:
It'll be good to provide some descriptions of what spec and spec.keys are in the constructor.